tasks/main.yml

```diff
@@ -91,7 +98,7 @@
     src: "server-{{ humio_version }}.jar"
     dest: "/usr/lib/humio/server-{{ humio_version }}.jar"
   notify: Restart Humio
-  when: humio_mirror == 'master'
```

Comment:
Was this a bug?

Reply:
Not sure, will ask @mwl what the intention was with master